jobs/integration/validation.py

```diff
@@ -252,7 +252,7 @@ async def test_auth_file_propagation(model, tools):
 async def test_status_messages(model):
     """Validate that the status messages are correct."""
     expected_messages = {
-        "kubernetes-master": "Kubernetes master running.",
+        "kubernetes-control-plane": "Kubernetes control-plane running.",
```

Comment:
Good, glad you've caught this one. I believe there are a few integration tests sprinkled through `charmed-kubernetes` org that need this adjustment